### PR TITLE
Added wiki for MLX90614 I2C Sensor

### PR DESCRIPTION
**Pull request by rahulOCR**
Written Wiki for PR -https://github.com/ArduPilot/ardupilot/pull/26886 submitted

### CODE REVIEW

File: common/source/docs/common-temperature-sensor.rst [hunk 1]
```diff
@@ -33,6 +34,8 @@ Attach the sensor to one of the autopilot's I2C ports using SDA and SCL pins. Th
 
 .. note:: The MAX31865 is an SPI device, which uses the same parameters and basic configuration procedure as I2C sensors but must be connected to an SPI port rather than an I2C port. Some autopilots do not expose an external SPI port, in which case the MAX31865 cannot be easily used.
 
+.. note:: The MLX90614 sensor is an infrared thermometer for non-contact temperature measurements. Its I2C address is 90 so TEMP1_BUS=90 for this type of sensor.
```
Comment (Hwurzburg) [inline common/source/docs/common-temperature-sensor.rst:37]:
```suggestion
.. note:: The MLX90614 sensor is an infrared thermometer for non-contact temperature measurements. Its I2C address is 90 so :ref:`TEMP1_BUS<TEMP1_BUS>` = 90 for this type of sensor.
```

**Review by Hwurzburg — CHANGES_REQUESTED** [review]

**Comment by Hwurzburg** [comment]
thanks for the wiki update!

File: common/source/docs/common-temperature-sensor.rst (hunk 1 above)
Comment (rahulOCR) [reply common/source/docs/common-temperature-sensor.rst:37]: accepted

**Comment by rahulOCR** [comment]
squashing the correction commit

**Pull request merged**